readthedocs/api/v3/serializers.py

```diff
@@ -59,10 +60,33 @@ class Meta:
         fields = []
 
 
+# TODO: decide whether or not include a `_links` field on the object
+#
+# This also includes adding `/api/v3/notifications/<pk>` endpoint,
+# which I'm not sure it's useful at this point.
```

Comment:
Not sure I follow this comment though. Why is this required?

Reply:
This comment seems to be incorrect here 😓. I remember writing this in the context of the `urls.py` file. I'm not sure what's doing here 🙃 

Re-reading it, maybe I'm referring here that since it adds the `_self` attribute pointing to `notifications-details` it will use `/api/v3/notifications/<pk>` 🤔